app/jobs/cache/browser_channels/responses_for_prefix.rb

```diff
@@ -38,18 +38,17 @@ def generate_brotli_encoded_channel_response(prefix:)
           wallet.paypal_wallet = paypal_wallet
           channel_response.wallets.push(wallet)
         end
-        if site_banner_lookup.publisher.bitflyer_connection.present?
+        if site_banner_lookup.publisher&.bitflyer_connection.present?
```

Comment:
I used to have this line here, which might help too 

site_banner_lookup.publisher.selected_wallet_provider_type == BITFLYER_CONNECTION

if we wanted to try that

Reply:
I'm reverting, this change wasn't necessary